pyiron_base/jobs/script.py

```diff
@@ -274,54 +274,22 @@ def set_input_to_read_only(self):
         super().set_input_to_read_only()
         self.input.read_only = True
 
-    def to_hdf(self, hdf=None, group_name=None):
-        """
-        Store the ScriptJob in an HDF5 file
-
-        Args:
-            hdf (ProjectHDFio): HDF5 group object - optional
-            group_name (str): HDF5 subgroup name - optional
-        """
-        super(ScriptJob, self).to_hdf(hdf=hdf, group_name=group_name)
-        with self.project_hdf5.open("input") as hdf5_input:
-            hdf5_input["path"] = self._script_path
-            hdf5_input["parallel"] = self._enable_mpi4py
-            self.input.to_hdf(hdf5_input)
-
-    def from_hdf(self, hdf=None, group_name=None):
-        """
-        Restore the ScriptJob from an HDF5 file
-
-        Args:
-            hdf (ProjectHDFio): HDF5 group object - optional
-            group_name (str): HDF5 subgroup name - optional
-        """
-        super(ScriptJob, self).from_hdf(hdf=hdf, group_name=group_name)
-        if "HDF_VERSION" in self.project_hdf5.list_nodes():
-            version = self.project_hdf5["HDF_VERSION"]
-        else:
-            version = "0.1.0"
-        if version == "0.1.0":
-            with self.project_hdf5.open("input") as hdf5_input:
-                try:
-                    self.script_path = hdf5_input["path"]
-                    gp = GenericParameters(table_name="custom_dict")
-                    gp.from_hdf(hdf5_input)
-                    for k in gp.keys():
-                        self.input[k] = gp[k]
-                except TypeError:
-                    pass
-        elif version == "0.2.0":
-            with self.project_hdf5.open("input") as hdf5_input:
-                if "parallel" in hdf5_input.list_nodes():
-                    self._enable_mpi4py = hdf5_input["parallel"]
-                try:
-                    self.script_path = hdf5_input["path"]
-                except TypeError:
-                    pass
-                self.input.from_hdf(hdf5_input)
-        else:
-            raise ValueError("Cannot handle hdf version: {}".format(version))
```

Comment:
Will this still load older GenericParameters then?  I don't know off hand when then 0.1.0 -> 0.2.0 bump for GenericParameters was, though.

Reply:
You introduced these changes back in October 2020 https://github.com/pyiron/pyiron_base/pull/34 this corresponds to `pyiron_base=0.2.X` . So I am fine with removing this functionality with `pyiron_base=0.8`

Reply:
Yup, that's long enough ago that we don't need to care.